tests/Owin.Security.OpenIdConnect.Server.Tests/OAuth2AuthorizationClientCredentialsGrantTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Owin.Testing;
+using Xunit;
+
+namespace Owin.Security.OpenIdConnect.Server.Tests {
+    public class OAuth2AuthorizationClientCredentialsGrantTests {
+
+        [Fact]
+        public async Task ClientCredentials_MissingCredentials_BadRequest() {
+            var server = TestServer.Create(app => {
+                app.Properties["host.AppName"] = "Owin.Security.OpenIdConnect.Server.Tests";
+                app.UseOpenIdConnectServer(new OpenIdConnectServerOptions {
+                    SystemClock = new TestClock()
+                });
+            });
+
+            var response = await server.HttpClient.PostAsync(
+                "https://example.com/connect/token",
```

Comment:
You should be able to use a relative path (by default, `http://localhost/` is implicitly used as a base URL by both the server and the client)